Project.toml

```
@@ -1,7 +1,7 @@
 name = "ITensorDocs"
 uuid = "421c5ee2-4dae-4799-bb38-a6a9d9332403"
 authors = ["ITensor developers <support@itensor.org> and contributors"]
-version = "0.1.0"
+version = "0.1.1"
```

```suggestion
version = "0.1.0"
```
This package isn't registered anywhere yet so no need to bump the version.